tests/apps/app_django.py

```diff
@@ -125,10 +125,17 @@ def complex(request):
     return HttpResponse('Stan wuz here!')
 
 
+def response_with_headers(request):
+    response = HttpResponse(content_type='')
+    response['X-Capture-This-Too'] = 'this too'
+    return response
```

Comment:
```suggestion
    return HttpResponse('Stan wuz here with headers!', headers={'X-Capture-This-Too': 'this too'})
```

Reply:
I think it is still safer if we cover the capture of a list of headers, and not just a single header. Although the `test_request_header_capture` tests with a list and currently that covers the same codepath, but that is an implementational detail which at least theoretically could change in the future.